pints/_mcmc/__init__.py

```diff
@@ -293,13 +293,60 @@ class MCMCController(object):
     method : class
         The class of :class:`MCMCSampler` to use. If no method is specified,
         :class:`HaarioBardenetACMC` is used.
+    transform : pints.Transform
+        A :class:`pints.Transform` that transform the model parameter space to
+        search space. If no transform is specified, search will be performed
+        in the model space.
     """
 
-    def __init__(self, log_pdf, chains, x0, sigma0=None, method=None):
+    def __init__(
```

Comment:
I think I'd prefer the keep `method` as the final argument

Reply:
Done